neutrino.go

```diff
@@ -158,7 +158,7 @@ type ServerPeer struct {
 	server         *ChainService
 	persistent     bool
 	knownAddresses map[string]struct{}
-	banScore       connmgr.DynamicBanScore
+	banScore       connmgr.DynamicBanScore // nolint:unused, false positive
```

Comment:
The ban scoring logic was removed in a previous commit, this should probably be removed there as well.

Reply:
Right, removed.